app/views/course/leaderboards/show.html.slim

```diff
@@ -1 +1,43 @@
-p To be implemented later
+= page_header t('course.leaderboard.index.header')
```

Comment:
Oh I see, you are using singular name. I would still prefer the short hand one, more conventional to me :-) , but I'm fine with both though.